setup.py

```diff
@@ -7,46 +7,48 @@
 from src.cirrus.plugins.docs.constants import DESC
 
 
-NAME = 'cirrus-docs'
```

Comment:
vscode did the single to double quote conversion automatically when I saved, so I just left it in